packages/batcher/runtime/src/index.ts

```diff
@@ -189,4 +189,7 @@ async function main(): Promise<void> {
   await runtime.run(gameInputValidator, batchedTransactionPoster, provider);
 }
 
-void main();
+main().catch(e => {
+  console.error(e);
+  process.exit(1);
```

Comment:
shouldn't this be caught under the `process.on('uncaughtException'` we have at the top of this file?

Reply:
Ah, I bet the problem is that an error during startup is still within 1000ms of the starting `lastCatchDate`, that's why it's being swallowed entirely.

Fixing that still gets us an error message + exit, although it's with the 0 success code. It is a simpler fix if you don't care about that.

```diff
-let lastCatchDate = Date.now();
+let lastCatchDate = Date.now() - MINIMUM_INTER_CATCH_PERIOD;
```